content/doc/developer/publishing/releasing.adoc

```diff
@@ -7,6 +7,69 @@ layout: developerguide
 
 . Make sure you have permissions to release the plugin. link:../requesting-hosting/[See this guide to learn more details]
 
-== Configuring Maven
+== Artifactory Credentials for Maven
 
-TODO
\ No newline at end of file
+You will need to tell Maven your credentials to access link:../artifact-repository[Artifactory].
+Log in to Artifactory, and from your profile, obtain the _encrypted password_.
+
+Create the file `~/.m2/settings.xml` (`~` representing your user home directory, e.g. `/home/yourname` or `C:\Users\yourname`) if needed, and make sure it contains the +<servers>+ element as shown below:
+
+----
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      http://maven.apache.org/xsd/settings-1.0.0.xsd">
+
+  <servers>
+    <server>
+      <id>maven.jenkins-ci.org</id> // <1>
+      <username>your_user_name_here</username>
+      <password>your_encrypted_password_here</password>
+    </server>
+  </servers>
+
+</settings>
+----
+<1> This is not a valid host name anymore, but still the ID used by default in the Jenkins plugin parent POM.
+    You may need to add additional `<server>` entries if your plugin POM overrides the inherited Maven `<distributionManagement>`, but this should be rare.
+
+== Set up GitHub to accept your SSH key
+
+Maven Release Plugin will automatically push to the repository when performing a release, so you need to link:https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/[set up GitHub to accept your SSH key].
+
+See the link:https://help.github.com/articles/connecting-to-github-with-ssh/[GitHub help on SSH] for more information.
+
+== Performing the release
+
+With GitHub and Maven credentials set up, performing a release should be as easy as running the following command:
+
+----
+mvn release:prepare release:perform
+----
+
+// Not sure about this:
+// NOTE: While it is be possible to specify the username and password on the command line, that would require your accounts on GitHub and the Jenkins community to match, and prevent you from using two factor authentication on GitHub.
+// Neither is a recommend practice.
```

Comment:
Feedback on this would be appreciated. Does someone have the definitive answer here?